file-store/src/main/java/org/keycloak/models/map/storage/file/writer/WritingMechanism.java

```diff
@@ -0,0 +1,95 @@
+package org.keycloak.models.map.storage.file.writer;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+import org.snakeyaml.engine.v2.common.FlowStyle;
+import org.snakeyaml.engine.v2.common.ScalarStyle;
+import org.snakeyaml.engine.v2.events.DocumentEndEvent;
+import org.snakeyaml.engine.v2.events.DocumentStartEvent;
+import org.snakeyaml.engine.v2.events.Event;
+import org.snakeyaml.engine.v2.events.ImplicitTuple;
+import org.snakeyaml.engine.v2.events.MappingEndEvent;
+import org.snakeyaml.engine.v2.events.MappingStartEvent;
+import org.snakeyaml.engine.v2.events.ScalarEvent;
+import org.snakeyaml.engine.v2.events.SequenceEndEvent;
+import org.snakeyaml.engine.v2.events.SequenceStartEvent;
+import org.snakeyaml.engine.v2.events.StreamEndEvent;
+import org.snakeyaml.engine.v2.events.StreamStartEvent;
+
+/**
+ * Mechanism which produces list of {@link Event}s. 
+ * 
+ * @author vramik
+ */
+public class WritingMechanism {
+
+    private final List<Event> events = new LinkedList<>();
+    private final ImplicitTuple implicitTuple = new ImplicitTuple(true, true);
+
+    public void startMapping() {
+        events.add(new MappingStartEvent(Optional.empty(), Optional.of("!!map"), true, FlowStyle.BLOCK));
```

Comment:
Can the constants from `org.snakeyaml.engine.v2.nodes.Tag` be used instead of literals?